iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/HealthCheckInstructionImplTest.java

```diff
@@ -76,11 +77,11 @@ void healthcheckCmd() {
     assertTextRange(tree.textRange()).hasRange(1, 0, 1, 29);
 
     assertThat(tree.isNone()).isFalse();
-    assertThat(tree.instruction()).isInstanceOf(CmdInstruction.class);
+    assertThat(tree.none()).isNull();
     assertThat(tree.instruction().getKind()).isEqualTo(DockerTree.Kind.CMD);
     assertThat(tree.options()).isEmpty();
 
-    List<SyntaxToken> cmdArguments = ((CmdInstruction) tree.instruction()).arguments().literals();
+    List<SyntaxToken> cmdArguments = tree.instruction().arguments().literals();
     assertThat(cmdArguments).hasSize(2);
     assertThat(cmdArguments.get(0).value()).isEqualTo("command");
     assertThat(cmdArguments.get(1).value()).isEqualTo("param");
```

Comment:
This may be an opportunity to write an helper in TestUtils to assert easily CMD content ;) If you think it make sense.